### PR TITLE
Encryption support

### PR DESCRIPTION
**Pull request by shomix**
This PR will:
- cleanup encryption package to only use `Uint8Array` instead of `Blob`, because `Uint8Array` is supported on both node and browser envs.
- Add `crypto` global support for node env.
- Add support for `encryptUpload` and `decryptUpload` functions to the `storage` SDK.
- Add an example on how to use the `storage` sdk to encrypt data -> https://github.com/spheronFdn/sdk/tree/encryption-v2/examples/storage-encryption

### CODE REVIEW

**Review by rekpero — APPROVED** [review]
looks really clean and clear

File: examples/storage-encryption/src/encrypt.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    accessControlConditions,
+    chain,
+    filePath,
+    litNodeClient: client,
+    configuration: {
+      name: bucketName,
+    },
+  });
+
+  console.log(uploadResponse);
```
Comment (ilij4) [inline examples/storage-encryption/src/encrypt.js:47]: is this intentional?

Comment (shomix) [reply examples/storage-encryption/src/encrypt.js:47]: Yeah.
This is just an example.

Comment (shomix) [reply examples/storage-encryption/src/encrypt.js:47]: if by this you meant about the console.log

Comment (ilij4) [reply examples/storage-encryption/src/encrypt.js:47]: @shomix yea, saw it there and at one more place so just wanted to check. Other than that looks good

**Review by ilij4 — APPROVED** [review]

**Pull request merged**